Make sure to set NV_DRIVER_DATE properly, using ChangeLog instead of the
output of git log, since most of the time, building won't happen from
the git repository.
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/configure.ac
+++ b/configure.ac
@@ -28,7 +28,7 @@ AC_INIT([xf86-video-nouveau],
         [xf86-video-nouveau])
 
 AC_DEFINE_UNQUOTED([NV_DRIVER_DATE],
-                   [$(echo -n \";git log |head -3|tail -1|tr -d '\n';echo -n \")],
+                   [$(echo -n \";cat $CURDIR/ChangeLog |head -3|tail -1|tr -d '\n';echo -n \")],
                    [Driver date])
 
 AC_CONFIG_SRCDIR([Makefile.am])
